perfmetrics/scripts/testing_on_gke/examples/dlio/unet3d-loading-test/templates/dlio-tester.yaml

```diff
@@ -16,7 +16,7 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: dlio-tester-{{ .Values.dlio.numFilesTrain }}-{{ .Values.dlio.recordLength }}-{{ .Values.dlio.batchSize }}-{{ .Values.scenario }}
+  name: dlio-tester-{{ .Values.scenario }}-{{ .Values.dlio.numFilesTrain }}-{{ .Values.dlio.recordLength }}-{{ .Values.dlio.batchSize }}
```

Comment:
The pod name is becoming quite big. IIUC, pod names cannot be greater than 63 characters.
I replaced some values from the sample configs which comes to:
dlio-unet3d-gcsfuse-no-file-cache-1000000-157286400-200. This is 56 characters long.
In case of replica sets with long names, Kubernetes only uses the first 58 characters and appends some random characters to help distinguish different pods. We could be hitting too close to that. Please verify if that's something we want.

Reply:
Yes, pod names are becoming longer, but it didn't cause any issues till this point.

This became an unavoidable problem in a later feature (supporting separate gcsfuse options for each workload) and I had to fix it there. It's fixed in dependent PR https://github.com/GoogleCloudPlatform/gcsfuse/pull/2348 .

> Kubernetes only uses the first 58 characters and appends some random characters to help distinguish different pods

I didn't encounter this.

Reply:
> IIUC, pod names cannot be greater than 63 characters

yes that's right.